src/components/FileItem/FileItem.styles.ts

```diff
@@ -0,0 +1,52 @@
+import { compressedVariantStyles, extendedVariantStyles } from './components';
+
+import type { BaseProps } from '@/types/BaseProps';
+
+export type FileItemConfig = BaseProps & {
+  state?: {
+    uploading?: BaseProps;
+    uploaded?: BaseProps;
+    replaceable?: BaseProps;
+    alert?: BaseProps;
+  };
+  inverted?: BaseProps;
+  invertedAlert?: BaseProps;
+  compressed?: BaseProps;
+  extended?: BaseProps;
+};
+
+export const defaultConfig = {
+  display: 'flex',
+  flexDirection: 'column',
+  gap: '$space-component-gap-small',
+  borderRadius: '$border-radius-large',
+  state: {
+    uploading: {
+      backgroundColor: '$color-interaction-neutral-subtle-normal',
+    },
+    uploaded: {
+      backgroundColor: '$color-interaction-default-subtle-normal',
+    },
+    replaceable: {
+      backgroundColor: '$color-interaction-default-subtle-normal',
+    },
+    alert: {
+      backgroundColor: '$color-interaction-alert-subtle-normal',
+    },
+  },
+  inverted: {
+    backgroundColor: '$color-interaction-background-formField',
+    borderWidth: '$border-width-100',
```

Comment:
I'm prette sure it should be alias value, not global: `$border-width-small`

Reply:
Yes. Changed.